oci/caps/utils.go

```diff
@@ -16,18 +16,6 @@ func init() {
 	if last == capability.Cap(63) {
 		last = capability.CAP_BLOCK_SUSPEND
 	}
-	if last > capability.CAP_AUDIT_READ {
-		// Prevents docker from setting CAP_PERFMON, CAP_BPF, and CAP_CHECKPOINT_RESTORE
-		// capabilities on privileged (or CAP_ALL) containers on Kernel 5.8 and up.
-		// While these kernels support these capabilities, the current release of
-		// runc ships with an older version of /gocapability/capability, and does
-		// not know about them, causing an error to be produced.
-		//
-		// FIXME remove once https://github.com/opencontainers/runc/commit/6dfbe9b80707b1ca188255e8def15263348e0f9a
-		//       is included in a runc release and once we stop supporting containerd 1.3.x
-		//       (which ships with runc v1.0.0-rc92)
-		last = capability.CAP_AUDIT_READ
-	}
```

Comment:
This will break dind (Docker 21 in Docker 20.10). I still think https://github.com/containerd/containerd/pull/5017 is the right approach.

Reply:
(Might be fine for runc >= rc94, though, as runc >= rc94 does not raise error on unknown caps https://github.com/opencontainers/runc/pull/2854)